dvc_http/__init__.py

```diff
@@ -136,6 +127,11 @@ async def get_client(
             sock_read=sock_read_timeout,
         )
 
+        if connector_kwargs:
+            kwargs["connector"] = aiohttp.TCPConnector(
+                loop=self.fs.loop, **connector_kwargs
+            )

```

Comment:
Since this is an async function, I don't think we should specify `loop`.
```suggestion
            kwargs["connector"] = aiohttp.TCPConnector(**connector_kwargs)
```

Reply:
IUnrelated but also see https://groups.google.com/g/python-tulip/c/yF9C-rFpiKk/m/tk5oA3GLHAAJ. `loop=` has been deprecated from most asyncio APIs.

Reply:
Removed  :pray: